src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,9 @@
+@mixin logo($size) {
```

Comment:
This file better name with '_'  in beginning.

More about naming and file structure in scss in this [video, timing 1:00](https://mate.academy/learn/css-advanced/sass#/video/931)